app/main.py

```diff
@@ -1,3 +1,58 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+import datetime
+
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    input_data = json.load(open("app/config.json", "r"))
+
+    fuel_price, customers, shops =\
+        input_data["FUEL_PRICE"], input_data["customers"], input_data["shops"]
+
+    customers = Customer.constructor(customers)
+    shops = Shop.list_constructor(shops)
+
+    for customers in customers:
+        print(f"{customers.name} has {customers.money} dollars")
+        best_shop = None
```

Comment:
You can move calculating of best shop to separate method or function

Reply:
The main.py module must contain only shop_trip function. 
![image](https://user-images.githubusercontent.com/32074944/219374376-9d412f75-74ca-4da6-a588-cf24272d148e.png)

Reply:
you could create a separate module for this